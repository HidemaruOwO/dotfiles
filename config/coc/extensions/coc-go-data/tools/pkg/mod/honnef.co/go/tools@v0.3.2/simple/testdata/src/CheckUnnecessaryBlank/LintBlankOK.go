package pkg

func fn() {
	var m map[int]int
	var ch chan int
	var fn func() (int, bool)

	x, _ := m[0] //@ diag(`unnecessary assignment to the blank identifier`)
	x, _ = <-ch  //@ diag(`unnecessary assignment to the blank identifier`)
	x, _ = fn()
	_ = x
}
