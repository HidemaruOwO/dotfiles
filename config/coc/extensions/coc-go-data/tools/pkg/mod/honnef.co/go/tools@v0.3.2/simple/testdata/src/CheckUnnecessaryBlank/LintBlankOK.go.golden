package pkg

func fn() {
	var m map[int]int
	var ch chan int
	var fn func() (int, bool)

	x := m[0] //@ diag(`unnecessary assignment to the blank identifier`)
	x = <-ch  //@ diag(`unnecessary assignment to the blank identifier`)
	x, _ = fn()
	_ = x
}
